packages/components/src/index.ts

```diff
@@ -15,3 +15,8 @@ export * from "./Icons"
 export * from "./InputSearch"
 export * from "./KaizenProvider"
 export * from "./Workflow"
+export * from "./Icons"
+// Migrated components
+// export * from "./ToggleSwitchField"
+// export * from "./GuidanceBlock"
+// export * from "./Scene"
```

Comment:
Delete these. A bit of a pain, but we'll have to do this later so we avoid merge conflicts.